README.md

```diff
@@ -281,9 +281,13 @@ And finally, you are ready to go! Or at least you should be...
 
 **Iris Mestres** | [Contact me][deliaBlue]
 
-### Junior Contributors
+**T**
 
+### Junior Contributors
 
+**Tassnime Reddahi: Let me infect you with some of my positive eneregy**
+ |[Tassnimered] <https://github.com/Tassnimered>
```

Comment:
Then this should be at the end, below you have the links of all of the reviewers placce your link where it fits so it is in alphabetical order:

without the "|" the following way: [Tassnimered] <https://github.com/Tassnimered>